src/Sylius/Bundle/OrderBundle/Command/RemoveExpiredCartsCommand.php

```diff
@@ -36,6 +36,11 @@ protected function configure()
      */
     protected function execute(InputInterface $input, OutputInterface $output)
     {
+        $expirationTime = $this->getContainer()->getParameter('sylius_order.cart_expiration_period');
+        $output->writeln(
+            sprintf('Command will remove carts that have been idle for <info>%s</info>.', $expirationTime)
```

Comment:
`Command will remove carts that have not been paid for` as it is a main purpose of this command